packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -30,16 +30,16 @@ import getUniquePresets from '../../../utils/getUniquePresets';
 
 function loadStory(storyId, post, restore, clearHistory) {
   const {
-    title: { raw: title },
+    title: { raw: title = '' } = '',
     status,
     slug,
     date,
     modified,
-    excerpt: { raw: excerpt },
+    excerpt: { raw: excerpt = '' } = '',
```

Comment:
```suggestion
    excerpt: { raw: excerpt = '' } = {},
```

Both work the same I suppose.